Main.hs

```diff
@@ -100,6 +101,12 @@ parsePackageSpec =
           Opts.short 't' <>
           Opts.metavar "URL" <>
           Opts.help "Used during 'update' when building URL. Occurrences of <foo> are replaced with attribute 'foo'."
+        )) <|>
+      (("fetcher",) <$> Opts.strOption
```

Comment:
I think you can simply add this as a wrapper around `-a fetcher=...` like https://github.com/nmattia/niv/pull/27/files#diff-98a3ddc5c474fd14c4ee9401c03197a1R121

Reply:
I didn't use a wrapper because I wanted to set a specific help message on this option.